.github/workflows/build-check.yml

```
@@ -0,0 +1,28 @@
+# https://www.viget.com/articles/host-build-and-deploy-next-js-projects-on-github-pages/
+
+name: Node.js CI
+
+on:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # https://github.com/actions/checkout
+      - uses: actions/checkout@v3
+        
+      # https://github.com/actions/setup-node
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: npm
+
+      - name: Build
+        run: |
+          npm ci
+          npm run build
+          npm run export
+          touch out/.nojekyll
```

Very minor comment here to fix missing newline before `EOF` :)
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
```suggestion
          touch out/.nojekyll

```

---

> Apologies for that mishap and thanks so much for this fix and the extra build on PR!!

No worries at all, and I appreciate your review and suggestions!